python/lsst/jointcal/jointcal.py

```diff
@@ -594,8 +600,11 @@ def _fit_photometry(self, associations, dataName=None):
             model = lsst.jointcal.ConstrainedPhotometryModel(associations.getCcdImageList(),
                                                              self.focalPlaneBBox,
                                                              visitOrder=self.config.photometryVisitOrder)
+            # potentially nonlinear problem, so we may need a line search to converge.
+            doLineSearch = True if self.config.allowLineSearch else False
```

Comment:
Why not just `doLineSearch = self.config.allowLineSearch`?

Reply:
Because I'm an idiot. Thanks.